cli/mqtt.py

```diff
@@ -13,10 +13,13 @@
 
 def mqtt_open(env):
     with env.config.open() as cfg:
-        printer = cfg.printers[0]
+        if env.printer >= len(cfg.printers):
+            log.fatal(f"Printer number {env.printer} out of range, max printer number is {len(cfg.printers)-1} ")
+            return
```

Comment:
This pattern (fatal log + return) has an easier pattern in this project: `log.critical()` exits the program with an error message (this is a project-specific override)